spring-petclinic-api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js

```diff
@@ -15,15 +15,26 @@ angular.module('ownerDetails')
 
         }).then(function (){
 
+            // add insurance details
             for(let i = 0; i < self.owner.pets.length; i ++){
                 let pet = self.owner.pets[i];
                 $http.get('api/insurance/pet-insurances/' + pet.id + '/').then(function (response){
-                    promiseDone[i] = true;
```

Comment:
Should we keep this?

Reply:
Simplified the whole thing, none of this is doing anything